webserver/views/details.html

```diff
@@ -115,13 +115,7 @@ <h1 class="uptime">
   </p>
 
   <div class="section events row">
-    <div class="col-md-6">
-    <%-partial ('partial/display_logs_list', {logs:critical_events, log_type:'Critical'})%>
-    </div>
-
-    <div class="col-md-6">
-    <%-partial ('partial/display_logs_list', {logs:warning_events, log_type:'Warning'})%>
-    </div>
+    <%- include partial/display_logs_list.html %>
```

Comment:
Why does this needed to be changed?
We added some duplication in display_log_list.html with it.